.github/workflows/integ_test.yml

```diff
@@ -1,7 +1,7 @@
 name: Integration Tests
 on:
   push:
-    branches: [main]
+    branches: [main, chore/update-xcode-ver-integ-tests]
```

Comment:
Is this intentional?

Reply:
i will revert this after the review/before pushing the changes. this was added to check if the changes were working.